Updating for style (#1524)
fluent/fluent-bit-docs: installation/aws-container.md

# Containers on AWS

> AWS maintains a distribution of Fluent Bit combining the latest official release with

Reviewer: ```suggestion
AWS maintains a distribution of Fluent Bit that combines the latest official release with
```

a set of Go Plugins for sending logs to AWS services. AWS and Fluent Bit are working
together to rewrite their plugins for inclusion in the official Fluent Bit
distribution.

## Plugins

The [AWS for Fluent Bit](https://github.com/aws/aws-for-fluent-bit) image contains Go
Plugins for:

> - Amazon CloudWatch as  `cloudwatch_logs`. See the

Reviewer: ```suggestion
- Amazon CloudWatch as `cloudwatch_logs`. See the
```

>   [Fluent docs](https://docs.fluentbit.io/manual/pipeline/outputs/cloudwatch) or the

Reviewer: Should these be "Fluent Bit docs" and not "Fluent docs"? Also, these are absolute links. Is that how the links are being handled in the docs?

Author: they mostly seem to be, and I haven't wanted to spend the time struggling with the (working) link structure when I could be fixing things that are actually broken or wrong.

Fighting with the links is a future problem.

Author: and yes, I fixed them to say "Fluent Bit docs"

  [Plugin repository](https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit).
- Amazon Kinesis Data Firehose as `kinesis_firehose`. See the
  [Fluent docs](https://docs.fluentbit.io/manual/pipeline/outputs/firehose) or the
  [Plugin repository](https://github.com/aws/amazon-kinesis-firehose-for-fluent-bit).
- Amazon Kinesis Data Streams as `kinesis_streams`. See the
  [Fluent docs](https://docs.fluentbit.io/manual/pipeline/outputs/kinesis) or the
  [Plugin repository](https://github.com/aws/amazon-kinesis-streams-for-fluent-bit).

These plugins are higher performance than Go plugins.

Also, Fluent Bit includes an S3 output plugin named `s3`.

- [Amazon S3](https://docs.fluentbit.io/manual/pipeline/outputs/s3)

## Versions and Regional Repositories

AWS vends their container image using
[Docker Hub](https://hub.docker.com/r/amazon/aws-for-fluent-bit), and a set of highly
available regional Amazon ECR repositories. For more information, see the
[AWS for Fluent Bit GitHub repository](https://github.com/aws/aws-for-fluent-bit#public-images).

The AWS for Fluent Bit image uses a custom versioning scheme because it contains

> multiple projects. To see what each release contains, see the [release notes on
> GitHub](https://github.com/aws/aws-for-fluent-bit/releases).

Reviewer: ```suggestion
multiple projects. To see what each release contains, see the
[release notes](https://github.com/aws/aws-for-fluent-bit/releases) on GitHub.
```
